content/post/altswiftui-travel/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Introducing AltSwiftUI in Rakuten Travel"
+date: 2021-07-06T18:18:13+09:00
+excerpt: "What is AltSwiftUI and how we introduced it in Rakuten Travel"
+timeToRead: 4
+authors:
+   - Kevin Wong
+draft: true
```

Comment:
please add a hero image

```yaml
hero: /post/altswiftui-trave/hero.png
```

Here are a couple of good sources of comercially usable images:

* https://unsplash.com/
* https://www.pexels.com/
* https://pixelmob.co/
* https://search.creativecommons.org/

please add license and attribution as required by the source.